src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
Actually - if it really helps with things (compilers really aren't "smart" enough to optimize it out?), it should be made to be a part of https://github.com/tux3/qTox/wiki/Coding, provided that this really helps, and no one sees it as a big problem.